zeroex/order.go

```diff
@@ -208,9 +210,9 @@ func SignOrder(signer ethereum.Signer, order *Order) (*SignedOrder, error) {
 }
 
 // SignTestOrder signs the 0x order with the local test signer
-func SignTestOrder(order *Order) (*SignedOrder, error) {
+func SignTestOrder(order Order) (*SignedOrder, error) {
```

Comment:
Is there a reason to use `Order` here instead of `*Order`? The latter is slightly more memory efficient and more closely matches the API of `SignOrder`.

Note that for `Order` struct inside `SignedOrder`, I wanted to change it to not be a pointer because `SignedOrder` is already passed around as a pointer pretty much everywhere. You don't gain any memory efficiency by having a pointer within a pointer. It's also easy to forget that embedded structs are pointer types they can be `nil`.

Reply:
Yeah agreed, reverted these changes.